README.md

```diff
@@ -42,15 +42,7 @@ object CentralRequirementsPlugin extends AutoPlugin {
     sonatypeProfileName := "com.yourdomain",
     // inform central who was explicitly involved in developing
     // this project. Note that this is *required* by central.
-    pomExtra in Global := {
-      <developers>
-        <developer>
-          <id>timperrett</id>
-          <name>Timothy Perrett</name>
-          <url>http://github.com/timperrett</url>
-        </developer>
-      </developers>
-    },
+    developers += Developer("timperrett", "Timothy Perrett", "hello@timperrett.com", url("http://github.com/timperrett"))
```

Comment:
Kindly remove my email from the plain text code here. It's not required by sonatype and i'd rather not get spam to my personal account.